src/services/login-handler.js

```diff
@@ -88,10 +88,6 @@ function LoginHandler({
       ).perform();
     }
 
-    if (!user) {
-      throw new Error();
-    }
-
```

Comment:
This is useless because if the user cannot be determined, it breaks before entering in that loop and the error is catched in the session.js file.